src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -186,144 +186,144 @@ public partial struct JsonReaderState
     }
     public static partial class JsonSerializer
     {
-        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("JSON serialization and deserialization might require types that cannot be statically analyzed. Use the overload that takes a JsonTypeInfo or JsonSerializerContext, or make sure all of the required types are preserved.")]
```

Comment:
The seemingly unrelated changes are caused by the `GenerateReferenceAssemblySource` target. It looks like many of the recent additions to this file were done by hand, so I'm incorporating the formatting favored by the generator in this change.